public_html/about.php

```diff
@@ -12,7 +12,8 @@
 // Parse YAML contributors file
 require_once("../Spyc.php");
 $contributors = spyc_load_file('../nf-core-contributors.yaml');
-$contributors_html = '<div class="card-deck">';
+$contributors_html = '<div id="contribution_map"></div><br>';
```

Comment:
Minor thing, but I have a pet-hate against using line breaks for spacing: CSS is better 😉 Especially as you're already adding CSS styles for this element.

Reply:
Fixed in new commit